docs/cel2/operators/run-node.md

```diff
@@ -340,8 +340,8 @@ Please reach out to our team on [Discord](https://chat.celo.org) in the [#celo-L
 
 - Container images:
   - [Celo L1 client](https://us-docker.pkg.dev/celo-org/us.gcr.io/geth-all:1.8.9)
-  - [op-geth](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth:celo-v2.0.0)
-  - [op-node](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-node:celo-v2.0.0)
+  - [op-geth](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth:celo-v2.1.0)
```

Comment:
This link seems to be broken

Reply:
It's still compiling, but yes, that should work in a few minutes

Reply:
Working now!